recipes/myst-nb/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "myst-nb" %}
+{% set version = "0.9.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/myst-nb-{{ version }}.tar.gz
+  sha256: 94e9e6c61033a9818a9e58f496043e92880b00882b39ae59d4762886ee63fe38
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - docutils >=0.15
+    - importlib-metadata
+    - ipython
+    - jupyter-cache >=0.4.0
+    - jupyter-sphinx >=0.2.4
+    - myst-parser >=0.12.4
+    - nbconvert >=5.6
+    - nbformat >=5.0
+    - python >=3.6
+    - pyyaml
+    - sphinx >=2,<4
+    - sphinx-togglebutton >=0.2.2
+
+test:
+  imports:
+    - myst_nb
+    - myst_nb.nb_glue
+  requires:
+    - pip
```

Comment:
?

Reply:
I removed those lines if you want to take another look